src/Plugin.Maui.Calendar/Shared/Controls/Calendar.xaml.cs

```diff
@@ -275,6 +279,8 @@ static void OnShownDateChanged(BindableObject bindable, object oldValue, object
 			calendar.UpdateDays(true);
 
 			calendar.OnShownDateChangedCommand?.Execute(calendar.ShownDate);
+
+			calendar.OnPropertyChanged(nameof(calendar.LocalizedYear));
```

Comment:
Can we avoid this line? Everything should be triggered from the property triggers